packages/component-library/src/HeroImage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { processStyle, createStyleBuilder, createBootstrap, StyleConfig as BaseStyleConfig } from './helpers';
+
+export interface Props {
+  id?: string;
+  className?: string;
+  children?: React.ReactNode;
+  style?: object;
+  url?: string;
```

Comment:
I think it makes sense to include this as part of the base component instead of the themed ones since it's going to be needed no matter what